solutions/standard/main.tf

```diff
@@ -72,6 +99,7 @@ locals {
 
 module "secrets_manager" {
   count                = var.existing_secrets_manager_crn != null ? 0 : 1
+  depends_on           = [ibm_iam_authorization_policy.kms_policy]
```

Comment:
this should depends on the time sleep, otherwise its pointless adding it

Reply:
resolved